### PR TITLE
feat(chat): add persistent current call panel

### PR DESCRIPTION
**Pull request by icepuma**
## Summary

- Add a persistent current-call panel in the app shell for the local active LiveKit call, with return, expand, settings, mute, camera, and hangup controls.
- Keep the current local call out of rediscovered call lists so the panel is the canonical control surface while the dock and DM panel still show other running calls.
- Preserve the existing split and expanded call layouts; this is a compact navigation/control surface for when the user is outside the owning channel or DM.
- Make the already-viewing state passive and keep toggle controls accessible with action labels that do not conflict with pressed state.
- Normalize both sides of the dock DM current-call comparison so mixed-case JIDs do not leave the current call duplicated in the rediscovery dock.

## Plan

- Keep the call UI XMPP-native: this PR only reads existing `$callState`, XEP-0353 DM activity, Muji/MUC presence, and LiveKit control state; it does not add non-XMPP call APIs.
- Add a Discord/Slack-style current-call panel in the app shell so an active LiveKit call stays visible and controllable after navigating away from its channel or DM.
- Reuse the existing call controls store for mute, camera, settings, hangup, and return-to-call routing instead of adding another call lifecycle path.
- Preserve the existing split and expanded call layout; the panel is a compact navigation/control surface, not a replacement for the call layout.
- Add focused rendering and event tests for active MUC calls, active DM calls, connecting state, return routing, accessibility, duplicate filtering, mixed-case DM current-call filtering, and quick controls.
- Run chat package verification before marking ready.

## XEP Review

- Reviewed local `xeps/xep-0353.xml` for Jingle Message Initiation and refreshed DM call state.
- Reviewed local `xeps/xep-0272.xml` for Muji presence semantics.
- Reviewed local `xeps/xep-0483.xml` for SFU/online-meeting context.
- No non-XMPP API was added.

## Verification

- `bun test tests/call-activity-dock.test.ts`
- `bun run lint`
- `bun test`
- `bun run build`
- `git diff --check`

## Review Notes

- Greptile flagged a mixed-case DM JID comparison in `CallActivityDock`; fixed by normalizing both sides and covering the comparison.
- Adversarial call-state/protocol review found no actionable issues.
- Adversarial frontend/accessibility review found duplicate current-call surfaces, conflicting toggle accessibility labels, and an active “Viewing” chip; all were fixed and covered.
- No dev server was started.
- No knip suppressions were added.

### CODE REVIEW

File: chat/src/components/calls/CallActivityDock.vue [hunk 1]
```diff
@@ -61,7 +65,23 @@ const entries = computed(() => buildCallActivityDockEntries({
...
+
+function isCurrentCallEntry(entry: CallActivityDockEntry): boolean {
+  const current = callState.value;
+  if (entry.kind === "channel") {
+    if (current.phase !== "active" && current.phase !== "muc-pending") return false;
+    if (current.kind !== "muc") return false;
+    return normalizeMucCallRoomJid(entry.roomJid) === normalizeMucCallRoomJid(current.peer);
+  }
+  if (current.phase !== "active" || current.kind !== "dm") return false;
+  return entry.peerJid === barePeerJid(current.peer).toLowerCase();
```
[the remark this answers is not in the document]
Comment (icepuma) [reply chat/src/components/calls/CallActivityDock.vue]: Fixed in `3c819eab` by normalizing `entry.peerJid` before comparing it with the active DM call peer, and added test coverage for the comparison path.
